scripts/zones/Castle_Oztroja/MobIDs.lua

```diff
@@ -41,4 +41,38 @@ YAA_HAQA_THE_PROFANE_PH =
     [17395953] = 17395954, -- -32.302 -16.250 -139.169
 }
 
+OZ_HANDLE_TABLE =
+{
+    [0] = {[17396196] = 8, [17396197] = 8, [17396198] = 8, [17396199] = 8},
+    [1] = {[17396196] = 8, [17396197] = 9, [17396198] = 9, [17396199] = 9},
+    [2] = {[17396196] = 9, [17396197] = 8, [17396198] = 9, [17396199] = 9},
+    [3] = {[17396196] = 9, [17396197] = 8, [17396198] = 8, [17396199] = 9},
+    [4] = {[17396196] = 9, [17396197] = 8, [17396198] = 9, [17396199] = 8},
+    [5] = {[17396196] = 8, [17396197] = 8, [17396198] = 9, [17396199] = 8},
+    [6] = {[17396196] = 9, [17396197] = 9, [17396198] = 8, [17396199] = 9},
+    [7] = {[17396196] = 9, [17396197] = 9, [17396198] = 9, [17396199] = 8},
+    [8] = {[17396196] = 8, [17396197] = 8, [17396198] = 9, [17396199] = 9},
```

Comment:
unless i read the other scripts wrong, these seem to be animation ids?
if so, we should use ANIMATION_NAME enums from `status.lua` instead of [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants)